BREEZE.rst

```diff
@@ -290,165 +264,170 @@ It is as easy as copy&pasting this line into your code:
+To build documentation in Breeze, use the ``-O``, ``--build-docs`` command:
 
-Cleanup can be performed with ``docker system prune`` command.
-Make sure to `Stop Breeze <#stopping-breeze>`_ first with ``./breeze --stop-environment``.
+.. code-block:: bash
 
-If you run into disk space errors, we recommend you prune your docker images using the
-``docker system prune --all`` command. You might need to restart the docker
-engine before running this command.
+     ./breeze --build-docs
 
-You can check if your docker is clean by running ``docker images --all`` and ``docker ps --all`` - both
-should return an empty list of images and containers respectively.
+Results of the build can be found in the ``docs/_build`` folder. 
 
-If you are on Mac OS and you end up with not enough disk space for Docker you should increase disk space
-available for Docker. See `Prerequsites <#prerequisites>`_.
+Often errors during documentation generation come from the docstrings of auto-api generated classes. 
+During the docs building auto-api generated files are stored in the ``docs/_api`` folder. This helps you easily identify the location the problems with documentation originated from.
 
 Troubleshooting
 ---------------
 
-If you are having problems with the Breeze environment - try the following (after each step you
-can check if your problem is fixed)
+If you are having problems with the Breeze environment, try the steps below. After each step you
+can check whether your problem is fixed.
 
-1. Check if you have enough disks space in Docker if you are on MacOS.
-2. Stop Breeze - use ``./breeze --stop-environment``
-3. Delete ``.build`` directory and run ``./breeze --force-pull-images``
-4. `Clean up docker images <#cleaning-up-the-images>`_
-5. Restart your docker engine and try again
-6. Restart your machine and try again
-7. Remove and re-install Docker CE and try again
+1. If you are on macOS, check if you have enough disk space for Docker.
+2. Stop Breeze with ``./breeze --stop-environment``.
+3. Delete the ``.build`` directory and run ``./breeze --force-pull-images``.
+4. `Clean up Docker images <#cleaning-up-the-images>`_.
+5. Restart your Docker Engine and try again.
+6. Restart your machine and try again.
+7. Re-install Docker CE and try again.
 
-In case the problems are not solved, you can set VERBOSE variable to "true" (`export VERBOSE="true"`)
-and rerun failing command, and copy & paste the output from your terminal, describe the problem and
-post it in [Airflow Slack](https://apache-airflow-slack.herokuapp.com/) #troubleshooting channel.
+In case the problems are not solved, you can set the VERBOSE variable to "true" (``export VERBOSE="true"``),
+rerun the failed command, copy-and-paste the output from your terminal to the _`Airflow Slack <https://apache-airflow-slack.herokuapp.com/>`_  #troubleshooting channel and add the problem description.
 
 
-Using Breeze for other tasks
-============================
+Testing in Breeze
```

Comment:
to me it sounds like we have lots of tests/checks/hooks for Airflow and all of them are chaotically described in different places now. I would recommend aggregating this info, for example, in the contribution guide and providing links to corresponding chapters for details. what do you think?

Reply:
Yes. Sounds like a good idea!

Reply:
I will merge the change in CONTRIBUTING.rst and update this change, so that you can move things to the already changed CONTRIBUTING.rst

Reply:
I merged the changes now, so you are free to move things between CONTRIBUTING.rst and BREEZE.rst @efedotova :).